src/qt/peertablemodel.cpp

```diff
@@ -179,9 +179,9 @@ QVariant PeerTableModel::data(const QModelIndex &index, int role) const
         assert(false);
     } else if (role == Qt::TextAlignmentRole) {
         switch (column) {
-        case NetNodeId:
         case Address:
             return {};
+        case NetNodeId:
```

Comment:
In e14d79f11eaf208b81fc90216304e1e0ea3d673e

Why move the position of `NetNodeId` here? In the other areas of the code, `NetNodeId` always comes before `Address` in our switch statements. I don't think we should move it here.

You can keep it where it originally is and do:

```suggestion
        case NetNodeId:
            return QVariant(Qt::AlignCenter);
```

Reply:
Generally I prefer to propose the smaller, simpler diff. The order doesn't matter here.

Reply:
It's not much of a difference, plus this makes it easier to reason about. You are explicitly stating that you are making it `AlignCenter`. I'd prefer we keep the same order as our other switch statements, but it's not a strong opinion.

Reply:
Always keeping the same order of enum elements in `switch` statement could decrease mental burden for some ppl :)

Reply:
Ditto, diff is only for "once" (for review), while people will read this switch multiple times in the future, so I'm also for same ordering.